src/Raven.Server/Documents/TimeSeries/TimeSeriesPolicyRunner.cs

```diff
@@ -452,7 +452,18 @@ private static bool RequiredForNextPolicy(DocumentsOperationContext context, Tim
                     return false;
 
                 if (nextEnd < to)
+                {
+                    var reader = tss.GetReader(context, docId, next.GetTimeSeriesName(raw), DateTime.MinValue, DateTime.MaxValue);
+                    if (nextStats == default && reader.AllValues(includeDead: true).Any())
```

Comment:
`reader.AllValues(includeDead: true).Any()` will materialize all entries. 
Can we use `reader.First()` instead?

BTW, having a live entry here is an invalid state, right?